### PR TITLE
RANSAC Segmentation

### PR DESCRIPTION
**Pull request by Reschivon**
No docs yet, need readme and testing

## Brief Description

TODO hehe

## Related Issues

Closes ...

Opens ...

## Tasks

- [ ] Finished all sub-tasks (from issues)
- [ ] Updated Documentation
- [ ] Tests written/updated and implemented to CI

### CODE REVIEW

**Comment by alanssitis** [comment]
do we still wanna use ransac since patchwork seems to work somewhat well?

**Comment by Reschivon** [comment]
looks like ransac is better with the improvements i made to it

File: src/amp_kart_segmentation/src/segmentation.cpp [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+        publish_cloud->header.stamp = input->header.stamp;
+        publish_cloud->header.frame_id = input->header.frame_id;
+
+        // Publish
+        cloud_pub->publish(std::move(publish_cloud));
+
+        // } catch(...) {
+        //     // if not nough points
+        //     RCLCPP_ERROR(get_logger(), "Encountered some error for this frame, doing nothing...\n");
+        // }
```
Comment (alanssitis) [inline src/amp_kart_segmentation/src/segmentation.cpp:187]: ditto

File: src/amp_kart_segmentation/src/segmentation.cpp [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+        declare_parameter("ring_gap", 10.);
+    }
+
+private:
+    // this function gets called every time new pcl data comes in
+    void cloudcb(const sensor_msgs::msg::PointCloud2::UniquePtr input)
+    {
+        // try{
+        // Get Parameters
+
```
Comment (alanssitis) [inline src/amp_kart_segmentation/src/segmentation.cpp:55]: remove unnecessary code

**Review by alanssitis — CHANGES_REQUESTED** [review]
Add it to `kart.launch.py` and update `README`.

**Comment by Reschivon** [comment]
@alanssitis ur precommit is missing a package for some reason

merge this pls

**Comment by alanssitis** [comment]
Close since #38 merged.

**Pull request closed without merging**